x-pack/plugins/ml/public/application/components/ml_inference/components/test_pipeline.tsx

```diff
@@ -0,0 +1,277 @@
+              <EuiFlexItem>
+                <EuiText>
+                  <h5>
+                    {i18n.translate(
+                      'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.subtitle.documents',
+                      { defaultMessage: 'Raw document' }
+                    )}
+                  </h5>
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem>
+                <EuiText>
+                  <h5>
+                    {i18n.translate(
+                      'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.subtitle.result',
+                      { defaultMessage: 'Result' }
+                    )}
+                  </h5>
+                </EuiText>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <EuiResizableContainer
+              direction={isSmallerViewport ? 'vertical' : 'horizontal'}
+              css={css`
+                min-height: calc(${euiThemeVars.euiSizeXL} * 10);
+              `}
+            >
+              {(EuiResizablePanel, EuiResizableButton) => (
+                <>
+                  <EuiResizablePanel grow hasBorder initialSize={50} paddingSize="xs">
+                    <CodeEditor
+                      languageId="json"
+                      options={{
+                        automaticLayout: true,
+                        lineNumbers: 'off',
+                        tabSize: 2,
+                      }}
+                      value={sampleDocsString}
+                      onChange={onChange}
+                    />
+                  </EuiResizablePanel>
+
+                  <EuiResizableButton />
+
+                  <EuiResizablePanel grow={false} hasBorder initialSize={50} paddingSize="xs">
+                    <EuiCodeBlock language="json" isCopyable className="reviewCodeBlock">
+                      {simulatePipelineError
+                        ? JSON.stringify(simulatePipelineError, null, 2)
+                        : simulatePipelineResult
+                        ? JSON.stringify(simulatePipelineResult, null, 2)
+                        : '{}'}
+                    </EuiCodeBlock>
+                  </EuiResizablePanel>
+                </>
+              )}
+            </EuiResizableContainer>
+          </EuiFlexItem>
+          <EuiSpacer />
+          <EuiFlexItem>
+            <EuiFlexGroup gutterSize="s" alignItems="center">
+              <EuiFlexItem grow={false}>
+                <div>
+                  <EuiButton
+                    onClick={simulatePipeline}
+                    disabled={sampleDocsString === '' || !isValid}
+                  >
+                    {i18n.translate(
+                      'xpack.ml.trainedModels.content.indices.pipelines.addInferencePipelineModal.steps.test.runButton',
+                      { defaultMessage: 'Simulate Pipeline' }
```

Comment:
```suggestion
                      { defaultMessage: 'Simulate pipeline' }
```

Reply:
Updated in 64ba3ac4ec279eb78f5f8a97d01005991ae76219